### ISSUE TITLE
post_cutout only writes first 256x256x16 pixels to image

### ISSUE THREAD

**Issue by kkutten1** [openconnectome/ndio#229]
| ndio version | os | python |
| --- | --- | --- |
| 1.1.3 | Ubuntu 14.04 | 2.7.6 |

This may be related to #200...

When posting an 655x815x1407 array to a 655x815x1408 image, the the post_cutout function only writes the first 256x256x16 pixels.  Furthermore it returns True despite failing to write all of the data.  DM me if you need the 'array.npz' file or the actual name of 'myToken'

Example code:

``` python
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import numpy as np
import ndio.remote.neurodata as neurodata

token = "myToken"
channel = "annotation"
array = np.load('array.npz')['arr_0']

nd = neurodata()
print("Array Size = " + str(array.shape))
print("Image Size = " + str(nd.get_image_size(token, 5)))
offset = nd.get_image_offset(token, 5)
print("Image Offset = " + str(offset))

returnValue = nd.post_cutout(token, channel, offset[0], offset[1], offset[2], data=array, resolution=5)
print("post_cutout returns " + str(returnValue))
```

Expected result: 
All pixels are written

Result:
Code prints

```
Array Size = (655, 815, 1407)
Image Size = [655, 815, 1408] 
Image Offset = [0, 0, 1] 
```

In http://openconnecto.me/ocp/ca/myToken/annotation/xy/5/0,655/0,815/1/ only first 256x256x16 pixels are written.

![result](https://cloud.githubusercontent.com/assets/9724840/17002137/f6f1f0da-4e97-11e6-95e9-fe48ee1c0216.png)

**Comment by wrgr** [comment]
Yea, this is an open issue - feel free to pull request.  It's on the stack.  :)

**Comment by jovo** [comment]
@wrgr where is the open issue? i couldn't find it?

**Comment by wrgr** [comment]
Referenced in Kwame's post - https://github.com/neurodata/ndio/issues/200

This did work and now seems to fail.  Needs debug and a unit/integration test.

**Comment by wrgr** [comment]
my guess is here:  https://github.com/neurodata/ndio/blob/master/ndio/remote/neurodata.py#L733

Likely not something I'll get to for at least a week.  We need to first deal with this, which generates incorrect data even with raw restful calls:  https://github.com/neurodata/ndstore/issues/303

**Comment by alexbaden** [comment]
This should be fixed in the ab-allen branch. I have another quick fix to add, then I'll do a PR.

**Comment by j6k4m8** [comment]
Resolved in #236, `pip install -U ndio` (version 1.1.5) to get the new code!
